interfaces/mount/change.go

```diff
@@ -42,6 +44,29 @@ type Change struct {
 	Action Action
 }
 
+// Perform executes the desired mount or unmount change.
```

Comment:
Is the mount backend the correct place for this then? I mean, it is topically similar, but functionally in the codebase, it is different....